Cargo.toml

```diff
@@ -81,7 +81,7 @@ human-repr = "1"
 human_bytes = "0.4"
 humantime = "2"
 indexmap = { version = "2", features = ["serde"] }
-indicatif = { version = "0.17", features = ["tokio"] }
```

Comment:
Since `cargo.lock` is committed and updated, I'd suggest not changing `Cargo.toml` to reduce commits to this file